src/renderer/versions.ts

```diff
@@ -15,37 +15,19 @@ import { normalizeVersion } from '../utils/normalize-version';
  * @param {Array<RunnableVersion>} knownVersions
  * @returns {string}
  */
-export function getDefaultVersion(
-  knownVersions: Array<RunnableVersion> = [],
-): string {
-  const ls = localStorage.getItem('version');
-
-  if (
-    ls &&
-    knownVersions &&
-    knownVersions.find(({ version }) => version === ls)
-  ) {
-    return ls;
-  }
-
-  // Self-heal: Version not formated correctly
-  const normalized = ls && normalizeVersion(ls);
-  if (normalized) {
-    if (
-      knownVersions &&
-      knownVersions.find(({ version }) => version === normalized)
-    ) {
-      return normalized;
-    }
-  }
```

Comment:
I've removed this 'self-heal' code because the only way this code path would get exercise if someone last used Electron Fiddle before we started saving normalized versions in LocalStorage, and is now starting the app for the first time since then. In this case their previously-selected version will have fallen off the main page due to being obsolete, so choosing `latestStable` is actually a safer choice.